instagram/src/components/PostContainer/PostPage.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import PostContainer from './PostContainer'
+
+
+class PostPage  extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            ummyDataList: [],
```

Comment:
Be very careful with your spelling!

Reply:
Gosh, that should be dummyDataList. I should have just used shorter variable names.